src/main/java/kr/syeyoung/dungeonsguide/chat/ChatProcessor.java

```diff
@@ -76,6 +88,15 @@ public void onTick(TickEvent.ClientTickEvent clientTickEvent) {
                     minimumNext = System.currentTimeMillis() + 200;
                     DungeonsGuide.sendDebugChat(new ChatComponentText("Sending " + tuple.getFirst() + " Secretly"));
                 }
+
+                if(!reciveQueue.isEmpty()){
```

Comment:
Would there be better way of implementing this?

1. It doesn't feel right to do this in chatprocessor (chat processor processes chat, not makes chat)
2. To display stuff, make Feature like Warning or Alert features. (painful for rapid-prototyping, but that is the way to go in dg)

Reply:
Btw it's receive not recive

Reply:
Sending chat might as well be a util class, doesn't contain business logic, so I vote to not make it a  `AbstractFeature`

Reply:
ChatTransmitter class was good change, but still fix typos there